From 38244625c086442dd841c624b8a661681bdd91e2 Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Mon, 20 Aug 2018 22:57:43 +0300
Subject: [PATCH 37/67] fix "cannot locate symbol
 "_ZN7android16MediaBufferGroupC1Ev" referenced by
 "/system/vendor/lib/libwvm.so

---
 media/libmediaextractor/MediaBufferGroup.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/media/libmediaextractor/MediaBufferGroup.cpp b/media/libmediaextractor/MediaBufferGroup.cpp
index 222d530c6..3863e606f 100644
--- a/media/libmediaextractor/MediaBufferGroup.cpp
+++ b/media/libmediaextractor/MediaBufferGroup.cpp
@@ -230,3 +230,5 @@ void MediaBufferGroup::signalBufferReturned(MediaBufferBase *) {
 }
 
 }  // namespace android
+
+__strong_alias(_ZN7android16MediaBufferGroupC1Ev,_ZN7android16MediaBufferGroupC1Ej);
-- 
2.11.0

